src/test/java/ui/RegisterTest.java

```diff
@@ -0,0 +1,46 @@
+package ui;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import uiPages.BasePage;
+import uiPages.RegisterPage;
+import static org.testng.Assert.*;
+
+import static helpers.Utilities.getPropertyFromRegProp;
+
+
+
+public class RegisterTest extends BaseTest {
+
+    private RegisterPage registerPage = null;
+
+    private String email;
+    private String password;
+    private  String username;
+
+    public RegisterTest() {
+
+        this.username = getPropertyFromRegProp("username");
+        this.email = getPropertyFromRegProp("email");
+        this.password = getPropertyFromRegProp("password");
+    }
+
+
+
+    @BeforeMethod
+    void beforeMethod() {
+        this.registerPage = new RegisterPage(getDriver());
+    }
+
+    @Test
+    public void testSuccessfulRegistration() {
```

Comment:
The flow is incomplete. After you click register on the shop.demoqa app you are redirected to an external site.
Although the register flow takes the user to an external page, this external page needs to be implemented as well in the code. 
You need to create a page class for that external page (e.g Wordpress), implement the needed methods, and then use it in the registration flow test.
Add this class in a separate folder (not java/uiPages) because this is an external page (not related to the shop.demoqa app). For example create new package 'externalUIPages'. The java/uiPages refers only to the shop.demoqa app pages.

Reply:
I created a new package containing the page class for the external page.